src/aosm/azext_aosm/configuration_models/onboarding_nfd_base_input_config.py

```diff
@@ -30,6 +35,5 @@ def validate(self):
             raise ValidationError("version must be set")
         if "-" in self.version or "." not in self.version:
             raise ValidationError(
-                "Config validation error. Version should be in"
-                " format 1.1.1"
-        )
+                "Config validation error. Version should be in" " format 1.1.1"

```

Comment:
this feels wrong

Reply:
Well spotted. I just tested, and this is how Black chooses to reformat the file - weird!

I've fixed it (and actually improved the message too).